### ISSUE TITLE
Add Makefile instructions for disable bass

### ISSUE THREAD

**Issue by sodomon2** [djoslin0/sm64ex-coop#264]
I open this as it is something that is needed, at least when building on `musl libc` based distros, as `libbass` is a library compiled only for `glibc`.

It would be nice to make an instruction to disable compilation with the bass libraries, as was done with [discord-sdk](https://github.com/djoslin0/sm64ex-coop/blob/4f62baebcce44b16a2bd7b0ee903577767e31154/Makefile#L52C17-L52C17).

**Comment by Isaac0-dev** [comment]
we'd rather remove bass entirely and use an alternative thats open source. finding one thats suitable for the job is the step we're up to I believe

**Comment by robertkirkman** [comment]
[working example `Makefile` for how to disable BASS](https://github.com/robertkirkman/sm64ex-coop/blob/android/Makefile) (BASS is disabled by default with that Makefile, but can be enabled by setting `TARGET_FOSS=0`)  **I have tested it working with `musl`.**

does not function without a large amount of changes throughout the code, which are currently included in all the files adjacent to that `Makefile`.

Should be noted that disabling BASS the way I demonstrate currently causes unpredictable issues with many Lua mods that expect functioning BASS. Some Lua mods work as expected and simply don't play BASS sounds, while others may derail and interfere with gameplay.

**Comment by robertkirkman** [comment]
> we'd rather remove bass entirely and use an alternative thats open source. finding one thats suitable for the job is the step we're up to I believe

I believe [miniaudio](https://miniaud.io/docs/manual/index.html) to be suitable unless it has a limitation I haven't yet encountered. I am attempting a solution using it, but haven't finished.

**Comment by sodomon2** [comment]
> [working example `Makefile` for how to disable BASS](https://github.com/robertkirkman/sm64ex-coop/blob/android/Makefile) (BASS is disabled by default with that `Makefile`, but can be enabled by setting `TARGET_FOSS=0` and undoing some other changes) **I have tested it working with `musl`.**
> 
> does not function without a large amount of changes throughout the code, which are currently included in all the files adjacent to that `Makefile`.
> 
> Should be noted that disabling BASS the way I demonstrate currently causes unpredictable issues with many Lua mods that expect functioning BASS. Some Lua mods work as expected and simply don't play BASS sounds, while others may derail and interfere with gameplay, depending on what ways they attempt to use BASS.

Yes, compiling with that `Makefile` would be easy, but as @Isaac0-dev says, it would be much better to find an open source alternative to replace `bass` and thus avoid the problems you say by not being enabled to build with `bass` support, plus it would work on systems that do not use `glibc`.
